teams/teams-ps/teams/Set-TeamTargetingHierarchy.md

```diff
@@ -0,0 +1,66 @@
+---
+external help file: Microsoft.Open.Teams.CommonLibrary.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/powershell/module/teams/connect-microsoftteams
+schema: 2.0.0
+author: mbergamo
+ms.author: mbergamo
+ms.reviewer:
+---
+# Set-TeamTargetingHierarchy
+
+## SYNOPSIS
+
+Upload a hierarchy to the tenant. A tenant may only have 1 active hierarchy. Each Set-TeamTargetingHierarchy cmdlet call will overwrite the previous one.
+
+## SYNTAX
+
+### Get (Default)
+```
+Set-TeamTargetingHierarchyStatus [-FilePath <String>]
+```
+
+## DESCRIPTION
+
+A sample CSV file uses the following format:
```

Comment:
Keep in mind that markdown and HTML tags will not render properly in PowerShell.